SIT.Manager.Avalonia/ViewModels/PlayPageViewModel.cs

```diff
@@ -85,6 +83,18 @@ private string Translate(string key, params string[] replaces)
             QuickPlayCommand = new AsyncRelayCommand(async () => await ConnectToServer(true));
         }
 
+        /// <summary>
+        /// Exists only to fix bug when you are changing language and this specific textblock doesn't get updated.
+        /// </summary>
+        private void ConfigService_ConfigChanged(object? sender, ManagerConfig e) => QuickPlayText = Translate("PlayPageViewModelQuickPlayText");
```

Comment:
This could be swapped out for a direct lambda function since we never need to deregister from the event